Public/Src/Cache/ContentStore/Interfaces/FileSystem/AbsolutePathExtensions.cs

```diff
@@ -3,6 +3,7 @@
 
 using System.Diagnostics.ContractsLight;
 using System.IO;
+using BuildXL.Cache.ContentStore.Interfaces.Utils;

```

Comment:
Still needed?

Reply:
Nope. Good catch!

---
In reply to: [280151698](https://github.com/Microsoft/BuildXL/pull/220#discussion_r280151698) [](ancestors = 280151698)